_maps/RandomRuins/LavaRuins/lavaland_surface_seed_vault.dmm

```diff
@@ -9,19 +9,89 @@
 /turf/closed/wall/r_wall,
 /area/ruin/powered)
 "d" = (
+/obj/structure/table/wood,
+/obj/item/weapon/lighter,
+/obj/item/weapon/lighter,
+/obj/item/weapon/storage/fancy/rollingpapers,
+/obj/item/weapon/storage/fancy/rollingpapers,
+/obj/item/weapon/storage/fancy/rollingpapers,
+/obj/item/weapon/storage/fancy/rollingpapers,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"e" = (
+/obj/structure/table/wood,
+/obj/item/weapon/storage/box/disks_plantgene,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"f" = (
+/obj/machinery/plantgenes/seedvault,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"g" = (
+/obj/structure/table/wood,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"h" = (
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"i" = (
+/obj/structure/closet/crate/hydroponics,
+/obj/structure/beebox,
+/obj/item/weapon/melee/flyswatter,
+/obj/item/honey_frame,
+/obj/item/honey_frame,
+/obj/item/honey_frame,
+/obj/item/queen_bee/bought,
+/obj/item/clothing/head/beekeeper_head,
+/obj/item/clothing/suit/beekeeper_suit,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"j" = (
 /obj/structure/shuttle/engine/propulsion{
 	icon_state = "propulsion";
 	dir = 8
 	},
-/turf/closed/mineral/volcanic/lava_land_surface,
+/turf/open/floor/plating/asteroid/basalt/lava_land_surface,
 /area/lavaland/surface/outdoors)
-"e" = (
+"k" = (
+/obj/structure/closet/crate,
+/obj/effect/spawner/lootdrop/seed_vault,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"l" = (
+/obj/machinery/door/airlock,
+/turf/open/floor/plasteel/freezer{
+	baseturf = /turf/open/floor/plating/lava/smooth
+	},
+/area/ruin/powered)
+"m" = (
+/obj/structure/shuttle/engine/propulsion{
+	icon_state = "propulsion";
+	dir = 8
+	},
+/turf/open/space/basic,
```

Comment:
Yes this needs to be fixed & the whole map checked over to ensure there's no raw space tiles on the ruin.